src/assets/constants.js

```diff
@@ -27,6 +25,19 @@ import DepopGrey from './images/depop-grey.svg'
 import KLEKTGrey from './images/klekt-grey.svg'
 import eBayGrey from './images/ebay-grey.svg'
 
+export const apiURL = 'https://rkzmvt26dg.us-west-2.awsapprunner.com'
```

Comment:
now that we have a `.env` we should move this in there

Reply:
I don't think it's really `.env` worthy since all web api calls are easily visible in the network tab